Assignment5/GenericsHomework/Node.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+namespace GenericsHomework
+{
+    public class Node<T>
+    {
+        private T? _Value;
+        private Node<T>? _Next;

```

Comment:
```suggestion
        private T _Value;
        private Node<T> _Next;
```
You do not need to set these as nullable. If you set them in the constructor you won't have any warnings.